src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Random random = new Random();
+        int number = random.nextInt(100);
+        return new Ball(colorSupplier.getRandomColor(),number);
+    }
```

Comment:
We can add fields with type `Random` and `ColorSupplier` to our class, so we won't need to create instances every time